queue_services/entity-filer/src/entity_filer/filing_processors/transition.py

```diff
@@ -0,0 +1,51 @@
+# Copyright © 2020 Province of British Columbia
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""File processing rules and actions for the transition of a business."""
+from typing import Dict
+
+from entity_queue_common.service_utils import QueueException
+from legal_api.models import Business, Filing
+
+from entity_filer.filing_processors.filing_components import aliases, shares
+from entity_filer.filing_processors.filing_components.offices import update_offices
+from entity_filer.filing_processors.filing_components.parties import update_parties
+
+
+def process(business: Business, filing_rec: Filing, filing: Dict):
+    # pylint: disable=too-many-locals; 1 extra
+    """Process the incoming transition filing."""
+    # Extract the filing information for transition application
+    transition_filing = filing.get('transition')
+
+    if not transition_filing:
```

Comment:
You could use the walrus here,
if not transition_filing := filing.get('transition'):